mikrotik/resource_interface_list_member.go

```diff
@@ -0,0 +1,105 @@
+package mikrotik
+
+import (
+	"context"
+
+	"github.com/ddelnano/terraform-provider-mikrotik/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceInterfaceListMember() *schema.Resource {
+	return &schema.Resource{
+		Description: "Allows to define set of interfaces for easier interface management.",
+
+		CreateContext: resourceInterfaceListMemberCreate,
+		ReadContext:   resourceInterfaceListMemberRead,
+		UpdateContext: resourceInterfaceListMemberUpdate,
+		DeleteContext: resourceInterfaceListMemberDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"interface": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"list": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceInterfaceListMemberCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	c := m.(*client.Mikrotik)
+	r := dataToInterfaceListMember(d)
+	record, err := c.AddInterfaceListMember(r)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	d.SetId(record.Id)
+
+	return resourceInterfaceListMemberRead(ctx, d, m)
+}
+
+func resourceInterfaceListMemberRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	c := m.(*client.Mikrotik)
+	record, err := c.FindInterfaceListMember(d.Id())
+	if err != nil {
+		return diag.FromErr(err)
+	}
```

Comment:
Is there a reason this doesn't identify the `NotFound` error and set the resource's id to an empty string?

Reply:
missed that, sharp eye :)